src/app/overview/page.tsx

```diff
@@ -0,0 +1,127 @@
+import { BlurredBox } from "@/components/ui/blurred-box";
+import { Container } from "@/components/ui/container";
+import { Heading } from "@/components/ui/heading";
+import { StarDecoration } from "@/components/ui/star-decoration";
+import { Text } from "@/components/ui/text";
+import Star from "../../assets/star.svg";
+import styles from "./page.module.css";
+export default function OverviewPage() {
+  return (
+    <Container as="main" className={styles.mainContainer}>
+      <StarDecoration>
+        <Heading as="h1" className={styles.headLine}>
+          開催概要
+        </Heading>
+      </StarDecoration>
+
+      <BlurredBox className={styles.firstGroup}>
```

Comment:
セクションだとよりセマンティックだと思います
```suggestion
      <BlurredBox as="section" className={styles.firstGroup}>
```